### PR TITLE
REL-535838 | Updated README breaking changes table

### PR DESCRIPTION
**Pull request by TommyBigNose**

### CODE REVIEW

**Review by aarongilbert123 — APPROVED** [review]

**Review by jake-martin — APPROVED** [review]

**Review by calimetikcura — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
+## Overview
+Open Source Community: Integration testing is a software testing methodology used to test individual software components or units of code to verify their interaction. These components are tested as a single group or organized in an iterative manner.
+Relativity Integration Test Helpers was created to assist with writing best practice Integration Tests for Relativity applications. The framework can be used to test event handlers, agents or any workflow that combines agents and event handlers.
 
-TestHelpers v8 release is only compatible with Ninebark (12.0) and above.  Any previous version of TestHelpers will only be compatible with Relativity until Prairie Smoke, which will remove RSAPI.
+The framework will currently enable you to to proxy and create documents, create workspaces and create dbcontext with more functionality to be added in the future.
+
+## Compatibility
+TestHelpers v8 release is only compatible with Ninebark (12.0) and above. Any previous version of TestHelpers will only be compatible with Relativity until Prairie Smoke {INSERT VERSION NUMBER), which will remove RSAPI.
```
Comment (aarongilbert123) [inline README.md:9]: Should we just say 12.2 instead of INSERT VERSION NUMBER?

Comment (TommyBigNose) [reply README.md]: The power of getting distracted while doing something! Fixed haha

**Review by aarongilbert123 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
+## Overview
+Open Source Community: Integration testing is a software testing methodology used to test individual software components or units of code to verify their interaction. These components are tested as a single group or organized in an iterative manner.
+Relativity Integration Test Helpers was created to assist with writing best practice Integration Tests for Relativity applications. The framework can be used to test event handlers, agents or any workflow that combines agents and event handlers.
 
-TestHelpers v8 release is only compatible with Ninebark (12.0) and above.  Any previous version of TestHelpers will only be compatible with Relativity until Prairie Smoke, which will remove RSAPI.
+The framework will currently enable you to to proxy and create documents, create workspaces and create dbcontext with more functionality to be added in the future.
```
Comment (calimetikcura) [inline README.md:6]: "enable you to to proxy" remove extra to

**Review by calimetikcura — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (TommyBigNose) [reply README.md]: Oops, thanks!  Fixed

**Pull request merged**